### ISSUE TITLE
Move MachineLearningMetrics.jl into JuliaML

### ISSUE THREAD

**Issue by paulhendricks** [JuliaML/META#6]
@tbreloff Would there be any interest in moving MachineLearningMetrics.jl into JuliaML? I would even consider renaming it MLMetrics, which is more succinct.

**Comment by paulhendricks** [comment]
Link: https://github.com/paulhendricks/MachineLearningMetrics.jl

**Comment by tbreloff** [comment]
Yes... I think that's a good idea.  (I was going to suggest it, but didn't want to pressure you)

@Evizero do you have an opinion on how the package fits in?

Also MLMetrics sounds good and fits with the pattern.

Thanks @paulhendricks!

**Comment by Evizero** [comment]
This looks great! Thanks for putting in the effort. I am also in favour of the name MLMetrics. 

You mentioned that you followed sklearn. Did you by any means look at the code while doing that? If so you should include that in your license.

**Comment by paulhendricks** [comment]
I did not look at the source code; I merely drew inspiration from the API around y_true and y_pred and what type of metrics are offered. There are way more regression metrics and classification metrics in MachineLearningMetrics.jl than in sklearn.metrics. 

I took most of the classification metrics from here: https://en.wikipedia.org/wiki/Precision_and_recall

**Comment by paulhendricks** [comment]
I certainly have no problem including them in the license though.

**Comment by Evizero** [comment]
I don't think that will be necessary then. I'm just covering the bases here :-).

I think you package would be a great addition

**Comment by paulhendricks** [comment]
@Evizero Awesome! 

What does the process of moving MLMetrics into JuliaML look like?

**Comment by Evizero** [comment]
it's already registered under MachineLearningMetrics, isn't it? In this case I don't think we even can rename it.

**Comment by Evizero** [comment]
Ok, so I invited you to be part of the team. Welcome!

Concerning the move: Two steps basically. First you move it to JuliaML (which is under the repository settings somewhere) and then you need to change the URL entry in METADATA to point at the new location (and submit a PR there with that change respectively)

**Comment by paulhendricks** [comment]
@Evizero @tbreloff I've moved the package into JuliaML. I have an issue open to see if we can change names since it's extremely new, very niche, and I would imagine current adoption is next to none. If that's not possible, we'll just change the url entry in metadata to point at the new location in JuliaML.

**Comment by Evizero** [comment]
nice

**Comment by paulhendricks** [comment]
Yup. See here for open issue: https://github.com/JuliaLang/METADATA.jl/issues/4951
